.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,12 +6,6 @@ title: ''
 labels: 's: unverified, type: bug'
 assignees: ''
 ---
-<!--
```

Comment:
I'm not sure if we should remove this comment.
Maybe rephrase it a bit, but overall this surely catches some people who want to create issues when theres a question, even if we still have some that entirely ignore it.

Reply:
I feel like the redirect solves this problem better than the comment does. This renders our GH hook description quite useless, if not removed
![image](https://user-images.githubusercontent.com/26532370/105996733-06853780-60ab-11eb-9c9b-438b1952fb03.png)

Additionally, we recently have converted questions that weren't super trivial to discussions, which might also be a valid approach going forward

Reply:
You are right in the assumption that the normal workflow would be:

"New issue" -> read what it says there -> evaluate whether to create an issue or go to the server -> create issue/go to the server.

But in most cases, I'm sure a lot of people just skip all the middle steps and just go directly to the first "bug report" because they cannot differentiate between a bug report and an issue on their side.

This is solely a further mitigation method on our part.

Reply:
I see that point, yeah, I'll put a short one-liner in place in hopes to deter some of the more persistent cases without sacrificing the entire GH webhook to it 👍